eq-author/src/index.js

```diff
@@ -52,7 +50,7 @@ const client = createApolloClient(link, cache);
 
 store = configureStore(history, client);
 
-if (window.Cypress && config.REACT_APP_FUNCTIONAL_TEST === "true") {
+if (window.Cypress && process.env.REACT_APP_FUNCTIONAL_TEST === "true") {
```

Comment:
We need this to be true for cypress tests. And we only build the UI once so it needs to be configurable from the docker run args.